sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_models.py

```diff
@@ -293,7 +293,6 @@ def __init__(
         transaction_id,  # type: str
```

Comment:
Change to types in place:
```suggestion
        transaction_id : str,
```

Reply:
Chatted offline - commented typings are no longer necessary as Python 2 is not supported anymore.